py/pythonhelpers.h

```diff
@@ -11,15 +11,15 @@
 #include <string>
 
 #if PY_MAJOR_VERSION >= 3
-#define FROM_STRING PyUnicode_FromString
 #define INITERROR return NULL
 #define MOD_INIT_FUNC(name) PyMODINIT_FUNC PyInit_##name(void)
 #else
-#define FROM_STRING PyString_FromString
 #define INITERROR return
 #define MOD_INIT_FUNC(name) PyMODINIT_FUNC init##name(void)
 #endif
 
+#define FROM_STRING PyUnicode_FromString
```

Comment:
Why did you make this change? Returning `unicode` in Py27 where it was previously `str` is an API-incompatible change.